### DISCUSSION TITLE
pnpm dev 启动失败

### DISCUSSION THREAD

**Discussion by liuyh208** [vbenjs/vue-vben-admin#1839] [Q&A]
下载代码后执行 npmp install 成功
执行 pnpm dev 启动调试失败，换了几个端口都不行，别的vite的项目正常
哪位大侠帮我看看，谢谢
![image](https://user-images.githubusercontent.com/2081365/164612369-46e3d53c-b0be-4986-9cc8-07c35c820743.png)

**Comment by vssir** [comment]
看提示应该是端囗权限问题，3106被占用？换其它再试，或者用管理员方式启动命令行来试
![image](https://user-images.githubusercontent.com/10388068/165198467-62adfb09-7997-44fe-b545-4eff399dbf1c.png)
